bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -882,6 +883,20 @@ def get_magnum_Matrix4_spot_a_T_b(self, a: str, b: str, tree=None) -> mn.Matrix4
         mn_transformation = mn.Matrix4.from_(rotation_matrix, translation)
         return mn_transformation
 
+    def get_sophus_SE3_vision_T_body(self, frame_tree_snapshot) -> sp.SE3:
+        """
+        Generates vision_T_body transform as a ROS Pose message from the FrameTreeSnapshot
+
+        Args:
+            frame_tree_snapshot (FrameTreeSnapshot): FrameTreeSnapshot from which to extract the vision_T_body transform
+
+        Returns:
+            sophus_SE3_vision_T_body (sp.SE3): sophus SE3 Pose containing the vision_T_body transform
+        """
+        vision_tform_body = get_vision_tform_body(frame_tree_snapshot)
+        sophus_SE3_vision_T_body = bd_SE3Pose_to_sophus_SE3(bd_se3=vision_tform_body)
+        return sophus_SE3_vision_T_body
+
```

Comment:
Try to make this generic - `get_sophus_SE3_vision_T_body(self, frame_tree_snapshot, a, b)`

Reply:
Updated to use `get_sophus_SE3_a_T_b(frame_tree_snapshot, rf.SPOT_WORLD_VISION, rf.SPOT_BODY)`